src/psyclone/tests/test_files/dynamo0p3/19.13_single_stencil.f90

```diff
@@ -43,12 +43,12 @@ program single_stencil
   implicit none
 
   type(field_type) :: f2_dofmap, f2, f3
-  integer(i_def)   :: f2_extent = 1, f3_stencil_size = 1
+  integer(i_def)   :: f2_extent = 1, f3_stencil_sizes = 1
 
   call invoke(                                                    &
        testkern_stencil_type(f2_stencil_map, f2, f2_extent,       &
```

Comment:
I think this was already wrong/incomplete but please could you add declarations for the kernel arguments. e.g. presumably `f2_stencil_map` is actually a field?

Reply:
Done, but that has to be the most confusing invoke statment ever.